content/docs/15.how-to-guides/index.md

```diff
@@ -3,7 +3,8 @@ title: How-to Guides
 icon: /docs/icons/tutorial.svg
 ---
```

Comment:
We need a way to hide all the sub pages in the side menu. It should only show "How-to Guides" at all times with the only way to change guide by going back to the how-to guide home page and using the filters/search

Reply:
`hideSidebar: true` doesn't quite solve this as it hides How-to Guides from the menu all together. Whereas when used for the getting started folder, it kept the "Getting Started" title visible but hide all the sub pages unless selected. We want similar to that but also hide sub pages when on them